core/templates/dev/head/pages/exploration_player/ExtractImageFilenamesFromStateService.js

```diff
@@ -105,6 +157,15 @@ oppia.factory('ExtractImageFilenamesFromStateService', [
     };
 
     return {
-      getImageFilenamesInState: _getImageFilenamesInState
+      getImageFilenamesInState: _getImageFilenamesInState,
+      getImageFilenamesInHints: function(state) {
+        var hintsHtml = _getHintsHtml(state);
+        return _extractFilepathValueFromOppiaNonInteractiveImageTag(hintsHtml);
+      },
+      getImageFilenamesInSolutions: function(state) {
+        var solutionHtml = _getHintsHtml(state);
+        return _extractFilepathValueFromOppiaNonInteractiveImageTag(
+          solutionHtml);
+      }
```

Comment:
Since we decided not to implement the priorities for solution/hints I think you can remove these.

Reply:
Done!